js/src/structured-data-blocks/how-to/components/HowTo.js

```diff
@@ -89,12 +86,20 @@ export default class HowTo extends Component {
 	 * @returns {void}
 	 */
 	setDurationText( text ) {
-		if ( text === "" ) {
-			text = this.getDefaultDurationText();
-		}
 		this.props.setAttributes( { durationText: text } );
 	}
 
+	/**
+	 * Sets the default duration text to describe the time the guide in the how-to block takes, when no duration text
```

Comment:
currently there's a bit of inconsistency between the terms "guide" and "instructions", if I remember correctly it was decided to change any occurrence of "guide"  to "instructions" /Cc @IreneStr